backend/infrahub/core/diff/calculator.py

```diff
@@ -86,6 +91,7 @@ async def calculate_diff(
         )
         node_limit = int(config.SETTINGS.database.query_size_limit / 10)
         fields_limit = int(config.SETTINGS.database.query_size_limit / 3)
+        properties_limit = int(config.SETTINGS.database.query_size_limit)
```

Comment:
`config.SETTINGS.database.query_size_limit` would already be an`int` so no need to convert it.